.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  main:
+    runs-on: ubuntu-18.04
```

Comment:
Any reason to not run on latest?

Reply:
It's mainly for the binaries (being installed in the action) that are depended upon by playwright; pinning the ubuntu distro makes it more predictable in the future. Plus `ubuntu-18.04` is LTS until 2023.